examples/admin_policy/example_policy/example_policy/skypilot_policy.py

```diff
@@ -190,6 +190,117 @@ def validate_and_mutate(
             task=task, skypilot_config=user_request.skypilot_config)
 
 
+class TokenBucketRateLimiter:
+    """Token bucket rate limiter."""
+
+    def __init__(self, capacity, fill_rate):
```

Comment:
nit: Can we have some basic docstr after L194 in terms of what these mean & how to set? That can help the users.